src/LogDocumentLinkProvider.ts

```diff
@@ -99,13 +112,10 @@ export class LogDocumentLinkProvider implements vscode.DocumentLinkProvider {
     }
 
     public convertPkgPathToFsPath(pkgPath: string) {
-        //remove any preceding file scheme
-        pkgPath = util.removeFileScheme(pkgPath);
-
-        //use debugRootDir if provided, or rootDir if not provided.
-        let rootDir = this.launchConfig.debugRootDir ? this.launchConfig.debugRootDir : this.launchConfig.rootDir;
-
-        let clientPath = path.normalize(path.join(rootDir, pkgPath));
-        return clientPath;
+        let mappedPath = this.getFileMap(pkgPath);
+        if (!mappedPath) {
```

Comment:
Can you add a few comments to the statements in this function? It's not clear which situations would cause a `mappedPath` to be undefined, and it's also not clear why we need to try looking for the `getAlternateBrsFileName` version of the `pkgPath` either.